integration/compliance_test.go

```diff
@@ -67,6 +67,7 @@ Global Flags:
   -s, --api_secret string   secret access key
       --debug               turn on debug logging
       --json                switch commands output from human-readable to json format
+      --nocache             turn off chaching
```

Comment:
typo
```suggestion
      --nocache             turn off caching
```

Reply:
Fixing this locally since I found more of these typos, nice catch! 💯